src/Generator/Generators/CodeGenerator.cs

```diff
@@ -395,6 +395,7 @@ public virtual bool VisitTypeAliasTemplateDecl(TypeAliasTemplate typeAliasTempla
     public static class Helpers
     {
         public static Regex RegexTag = new Regex(@"^(<|</)[a-zA-Z][\w\-]*?>?$");
+        public static Regex RegexCustomTag = new Regex(@"^_\S*");
```

Comment:
This regex doesn't really match a tag but rather a part of a comment command. Please rename accordingly.